### PR TITLE
Update mockit and Unit test version

### PR DESCRIPTION
**Pull request by dan-du-car**
Enable unit tests and update Junit and mockit dependencies' versions

### CODE REVIEW

File: jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPacketFunctionalTest.java [hunk 1]
```diff
@@ -35,14 +35,14 @@
...
    
-   @Capturing
-   DatagramSocket capturingDatagramSocket;
-   
-   @Capturing
-   DatagramPacket capturingDatagramPacket;
-   
-   @Capturing
-   Thread capturingThread;
+//   @Capturing
```
Comment (snallamothu) [inline jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPacketFunctionalTest.java:38]: Remove commented code if it's no longer needed.  Comment applies to commented out code in other files.

File: jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPointTest.java [hunk 2]
```diff
@@ -53,13 +53,13 @@ public void testStringConstructorCreatesAddress() {
             }
          };
 
-         new InetPoint("hostString123", 5, true);
+         new InetPoint("bah.com", 5, true);
```
Comment (snallamothu) [inline jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPointTest.java:56]: Why is "bah.com" hardcoded here?

**Review by snallamothu — CHANGES_REQUESTED** [review]

File: jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPointTest.java (hunk 2 above)
Comment (dan-du-car) [reply jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPointTest.java:56]: This is unit testing against a particular condition with a host name for unit testing InetPoint class.

File: jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPacketFunctionalTest.java (hunk 1 above)
Comment (dan-du-car) [reply jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPacketFunctionalTest.java]: It is addressed.

**Review by snallamothu — APPROVED** [review]

**Pull request merged**